serverless/admin_guide/serverless-kafka-admin.adoc

```diff
@@ -32,10 +32,23 @@ The `KnativeKafka` CR provides users with additional options, such as:
 
 include::modules/serverless-install-kafka-odc.adoc[leveloffset=+1]
 
-include::modules/serverless-kafka-broker-tls-default-config.adoc[leveloffset=+1]
-include::modules/serverless-kafka-broker-sasl-default-config.adoc[leveloffset=+1]
+[NOTE]
+====
+If you choose to enable SASL, Red Hat recommends to also enable TLS.
+====
+
+[id="serverless-kafka-admin-security"]
+== Security configuration for Knative Kafka
+
+Kafka clusters are generally secured by using the TLS or SASL authentication methods. You can configure a Kafka broker or channel to work against a protected Red Hat AMQ Streams cluster by using TLS or SASL.
```

Comment:
you can also do TLS with SASL

Reply:
Moved note just to the main intro section